zproject/default_settings.py

```diff
@@ -39,10 +39,17 @@
 
 # LDAP auth
 AUTH_LDAP_SERVER_URI = ""
+AUTH_LDAP_BIND_DN = ""
+AUTH_LDAP_USER_SEARCH: Optional["LDAPSearch"] = None
 LDAP_APPEND_DOMAIN: Optional[str] = None
 LDAP_EMAIL_ATTR: Optional[str] = None
-AUTH_LDAP_USERNAME_ATTR: Optional[str] = None
 AUTH_LDAP_REVERSE_EMAIL_SEARCH: Optional["LDAPSearch"] = None
+AUTH_LDAP_USERNAME_ATTR: Optional[str] = None
+AUTH_LDAP_USER_ATTR_MAP: Dict[str, str] = {
+    "full_name": "cn",
+}
```

Comment:
Why do we want a default value here, given that it's a required setting in `prod_settings_template.py`?

Reply:
As documentation of the value as a setting, and as a default in case we're not in prod.  I'd love if we could comment out the value in `prod_settings_template.py` but it's difficult to do that in a way that makes it straightforward to uncomment.

Reply:
OK, I'll add a comment noting that detail above it and merge.